### ISSUE TITLE
BUG: arrow keys not working while editing cell

### ISSUE THREAD

**Issue by mcanepa** [beekeeper-studio/beekeeper-studio#2242]
Double click to edit cell and cursor goes to the end of content. Then press arrow key to go left but the cursor doesn't move. I have manually move the mouse to desired position. (right arrow key doesn't work either)

**Comment by rathboma** [comment]
@azmy60 is the fix for this in master? I know you did a lot of work on the keyboard behavior

**Comment by azmy60** [comment]
No, this is new. Those were for trapping focus while navigating buttons in modals.
